app/models/setting.rb

```diff
@@ -58,4 +58,8 @@ def self.default_start_date
   def self.fb_reachable_period
     (ENV["FB_REACHABLE_DAY"] || 1).to_i
   end
+
+  def self.admin_view?
+    ENV["DASHBOARD_VIEW"].to_s == "private"
```

Comment:
@radinreth please consider using naming which links to what feature we're going to work on, in e.g: ENABLE_TG_DO_MONTHLY_REPORT or ENABLE_TG_MONTHLY_SCHEDULE_REPORT so that we can easily get what is this environment variable relevant to what feature in our software product.

Reply:
This variable is shared and also the variable that identify the difference between public dashboard and admin additionally I don't want to introduce new variable just for that purpose

Reply:
but your suggestion also make sense

Reply:
@radinreth we can use/reuse a variable if it's on the same meaning/definition for a different feature better use the right term so that we won't get ourself lost when we review it back